PacketNet.sln.DotSettings

```diff
@@ -0,0 +1,45 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Can you explain what this change is doing? Is the idea to somehow whitelist certain abbreviations to prevent Resharper from suggesting changing them? I think that would be good info to put into the commit body.

Reply:
Exactly as you say, I'll add it to the commit body. It eases future refactorizations and suggests to use some abbreviation consistently across the solution.